ui/src/index.css

```
@@ -165,14 +167,29 @@ main {
+  }
+
+  50% {
+    background-position: 100% 50%;
+  }
+
+  100% {
+    background-position: 0% 50%;
+  }
 }
 
 @keyframes heroShine {
```

Stylelint (17.7.0)

[error] Expected keyframe name "heroShine" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)